docs/themes/README.md

```diff
@@ -0,0 +1,61 @@
+# Themes
+
+> Themes for applying color in Carbon for IBM Security
+
+Also refer to [themes in Carbon](https://github.com/carbon-design-system/carbon/tree/master/packages/themes).
```

Comment:
I wonder about pointing folks to this link now, since the directions below (using a class name) versus what Carbon suggestions (scss build etc) are so different? (Also, related: #170)

Should we just leave this part out? There's a good explanation of themes in the "Usage" section below so that could be enough for now?

Reply:
Good point! I wanted folks to have access to the higher-level documentation for reference just in case, but it may be confusing or misleading to have it at the very beginning - I'll move this to the bottom of the file